packages/material-ui-system/src/createBox.js

```diff
@@ -10,7 +10,7 @@ function isEmpty(obj) {
 
 const useTheme = (defaultTheme) => {
   const contextTheme = React.useContext(ThemeContext);
-  return isEmpty(contextTheme) ? defaultTheme : contextTheme;
+  return (!contextTheme || isEmpty(contextTheme)) ? defaultTheme : contextTheme;
```

Comment:
`contextTheme` can be `null`, which breaks when invoking `isEmpty`, as it assumes a valid object is always passed.